docs/components/data_tables.md

```diff
@@ -0,0 +1,407 @@
+<!--docs:
+title: "Data tables"
+layout: detail
+section: components
+excerpt: "Data tables display information in a grid-like format of rows and columns."
+iconId: 
+path: /catalog/data-tables/
+api_doc_root: true
+-->
+
+# Data tables
+
+[Data tables](https://material.io/components/data-tables/#) display sets of data across rows and columns.
+
+![Data table with 5 rows: 1 header row, 4 rows, one column of checkboxes](assets/data_tables/data-table-hero.png)
+
+## Using data tables
+
+Data tables display information in a grid-like format of rows and columns. They organize information in a way that’s easy to scan, so that users can look for patterns and insights.
+
+Data tables can contain:
+
+* Interactive components (such as chips, buttons, or menus)
+* Non-interactive elements
+* Tools to query and manipulate data
+
+### Making data tables accessible
+
+Flutter's text field component APIs supports both label text and helper text for accessibility.
+
+For more guidance on writing labels, go to [our page on how to write a good accessibility label](https://material.io/design/usability/accessibility.html#writing).
+
+## Data tables
+
+* [Class description](https://api.flutter.dev/flutter/material/DataTable-class.html)
+* [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/data_table.dart)
+* [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/data_tables/normal)
+* [Demo site](https://gallery.flutter.dev/#/demo/data-table)
+
+![Demo table without Theme](assets/data_tables/data_table_no_theme.png)
+
+```dart
+
+
+class DataTableDemo extends StatefulWidget {
+  @override
+  _DataTableDemoState createState() => _DataTableDemoState();
+}
+
+class _DataTableDemoState extends State<DataTableDemo> {
+
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Data Tables'),
+      ),
+      body: ListView(
+        padding: const EdgeInsets.all(16),
+        children: [
+          PaginatedDataTable(
+            header: Text('Header Text'),
+            rowsPerPage: 4,
+            columns: [
+              DataColumn(label: Text('Header A')),
+              DataColumn(label: Text('Header B')),
+              DataColumn(label: Text('Header C')),
+              DataColumn(label: Text('Header D')),
+            ],
+            source: _DataSource(context),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _Row {
+  _Row(
+    this.headerA,
+    this.headerB,
+    this.headerC,
+    this.headerD,
+  );
+
+  final String headerA;
```

Comment:
Same comment about renaming these properties to `valueFoo`

Reply:
Done